lls/main.c

```diff
@@ -224,6 +253,34 @@ lls_scan(__attribute__((unused)) struct rte_timer *timer, void *arg)
 		lls_cache_scan(lls_conf, &lls_conf->nd_cache);
 }
 
+/* Send gratuitous ARP or ND announcement to fulfill LACP TX requirement. */
+static void
+lls_lacp_announce(__attribute__((unused)) struct rte_timer *timer, void *arg)
+{
+	struct lls_config *lls_conf = (struct lls_config *)arg;
+	struct net_config *net_conf = lls_conf->net;
+	if (lacp_enabled(net_conf, &net_conf->front)) {
+		if (iface_arp_enabled(net_conf, &net_conf->front))
+			xmit_arp_req(&net_conf->front,
+				(const uint8_t *)&net_conf->front.ip4_addr,
+				NULL, lls_conf->tx_queue_front);
+		else
```

Comment:
It would be nice to have an `else if (likely(iface_nd_enabled(...)))` followed by an `else rte_panic(...)` here and in analog code below just to quickly catch hard bugs when they show up.